harness/determined/searcher/_search_runner.py

```diff
@@ -191,11 +196,24 @@ def post_operations(
             searcherOperations=[op._to_searcher_operation() for op in operations],
             triggeredByEvent=event,
         )
-        bindings.post_PostSearcherOperations(
-            session,
-            body=body,
-            experimentId=experiment_id,
-        )
+        # TODO: Remove this temporary hack in favor of an actual solution around the issue described
+        # in https://determined-ai.slack.com/archives/C04645NHSP6/p1680201334658789
+        # Also verify that this is actually catching the error.
```

Comment:
shouldn't this be something that is verified before landing it?

Reply:
yeah, we just left that comment erroneously